autoware_launch/rviz/autoware.rviz

```diff
@@ -18,10 +18,6 @@ Panels:
     Splitter Ratio: 0.5
   - Class: tier4_localization_rviz_plugin/InitialPoseButtonPanel
     Name: InitialPoseButtonPanel
-  - Class: AutowareDateTimePanel
-    Name: AutowareDateTimePanel
-  - Class: rviz_plugins::AutowareStatePanel
-    Name: AutowareStatePanel
```

Comment:
Unnecessary change is included.
Could you check?

Reply:
Sorry, I'll fix it

Reply:
@rej55 
Fix https://github.com/autowarefoundation/autoware_launch/pull/308/commits/447b174a2f17fa911fb1fd9e98717cb283532eed